src/core/distributions/phylogenetics/tree/birthdeath/BirthDeathBurstProcess.cpp

```diff
@@ -433,8 +451,8 @@ double BirthDeathBurstProcess::lnQ(double t) const
         E_previous /= (2*birth);
 
         B = ((1.0 - 2.0*((1.0-burst)*E_previous+burst*E_previous*E_previous)) * birth + death ) /  A;
+//        B = ((1.0 - 2.0*E_previous) * birth + death ) /  A;
```

Comment:
This this the remnant of an experiment?

Reply:
yes, we found bugs in the theory and tested these. Last I remember is that this now works.